doc/user/content/sql/appendix-cluster-sizes.md

```diff
@@ -7,8 +7,40 @@ menu:
     weight: 900
 ---
 
+## Default Cluster Sizes
+
 {{% self-managed/materialize-cluster-sizes %}}
 
+## Custom Cluster Sizes
+
+When installing the Materialize Helm chart, you can override the [default
+cluster sizes and resource allocations](/sql/appendix-cluster-sizes/). These
+cluster sizes are used for both internal clusters, such as the `system_cluster`,
+as well as user clusters.
+
+{{< tip >}}
+
+In general, you should not have to override the defaults. At minimum, we
+recommend that you keep the 25-200cc cluster sizes.
+
+{{</ tip >}}
+
+```yaml
+operator:
+  clusters:
+    sizes:
+      <size>:
+        workers: <int>
+        scale: <int>
+        cpu_exclusive: <bool>
+        cpu_limit: <float>         # e.g., 6
+        credits_per_hour: <string> # e.g., "0.0"
+        disk_limit: <string>       # e.g., "93150MiB"
+        memory_limit: <string>     # e.g., "46575MiB"
+```
+
+{{< yaml-table data="best_practices/sizing_recommendation" >}}
```

Comment:
We also have some random blurbs scattered across (listing here)
- If spill-to-disk is not enabled: 1:8 ratio of vCPU to GiB memory
- If spill-to-disk is enabled (Recommended): 1:16 ratio of vCPU to GiB local instance storage

- 2:1 disk-to-RAM ratio with spill-to-disk enabled.
- 2:1 disk-to-RAM ratio with spill-to-disk enabled.

Once we settle on what's what, will rework into unified statements.

Reply:
>If spill-to-disk is not enabled: 1:8 ratio of vCPU to GiB memory
> If spill-to-disk is enabled (Recommended): 1:16 ratio of vCPU to GiB local instance storage

Where is this coming from? I'm pretty sure we always want 1:8. 

> 2:1 disk-to-RAM ratio with spill-to-disk enabled
sounds right

Reply:
> Where is this coming from? I'm pretty sure we always want 1:8.

https://github.com/MaterializeInc/materialize/tree/main/misc/helm-charts/operator#operational-guidelines